### DISCUSSION TITLE
Call for tags we need to add to sphinx-gallery

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#6377] [Ideas]
### Describe the feature you would like to be added.

In #6375 we added sphinx-tags in sphinx-gallery. In this thread, we call for tags that would be useful to add to sphinx-gallery.

### Links to VTK Documentation, Examples, or Class Definitions.

https://github.com/pyvista/pyvista/pull/6375

### Pseudocode or Screenshots

None

**Comment by tkoyama010** [comment]
We need
- [ ] CFD
- [ ] Texture
- [ ] FEA
- [ ] Image

**Reply by banesullivan** [reply]
IMO, `Image` is too vague as this could be inferred to mean `Texture` or `ImageData` or "example with an image"

**Comment by adeak** [comment]
- [ ] polydata are probably too common, but most of the other grid types
- [ ] animation

And my general recommendation is to add few tags at a time, but making sure that all relevant examples are tagged. Such a tag system is only useful if there aren't many false negatives.

And perhaps some automatic check could be made that all examples have a tag? This might be too over-engineered, but it's easy to forget about this when adding a new example.

**Reply by adeak** [reply]
[Now that I know about it](https://github.com/pyvista/pyvista/pull/6378#issuecomment-2229210390), it might suffice to add this information in the "add new example" example.
